### DISCUSSION TITLE
Got through waitlist but can't activate Copilot

### DISCUSSION THREAD

**Discussion by IngvarOlsen** [community/community#14905] [Copilot Conversations]
Hi, I got the mail that the wait was over, installed copilot for Visual Code, but when I logged in with Github it said that I needed to sign up for waitlist.
I guess it might be a bug? I did not promptly activate it though, got the mail back in 
29 Mar 2022, but haven't had the time to play around with it before now.

**Comment by davecheney** [comment]
Are you using VSCodium?

**Comment by FilipKufalov** [comment]
You should check if you use the same Github account for waitlist and VScode else It could be a bug
